policyengine_uk/variables/gov/hmrc/study_childcare_entitlement/care_to_learn_eligible.py

```diff
@@ -0,0 +1,41 @@
+from policyengine_uk.model_api import *
+
+
+class care_to_learn_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible for Care to Learn childcare support"
+    documentation = "Whether the person is eligible for Care to Learn support"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        # Only parents can be eligible, not children
+        is_parent = ~person("is_child", period)
+
+        # Check basic eligibility conditions
+        has_children = person.benunit.any(person("is_child", period))
+        p = parameters(period).gov.hmrc.study_childcare_entitlement
+        under_20 = person("age", period) < p.care_to_learn_age_eligible
```

Comment:
Avoid parameter values in variable names
```suggestion
        age_eligible = person("age", period) < p.care_to_learn_age_eligible
```